tasks/bake.js

```diff
@@ -483,18 +500,26 @@ module.exports = function( grunt ) {
 				if ( oldValue === undefined ) values[ forEachName ] = oldValue;
 				else delete values[ forEachName ];
 
-				return fragment;
+				content = fragment;
 
 			} else if( !forEachName ) {
 
 				processExtraBake( extraBake, filePath, destFile, values );
 
-				return linebreak + parse( includeContent, includePath, destFile, values );
+				content = linebreak + parse( includeContent, includePath, destFile, values );
 
 			} else {
 
+				content = "";
+			}
+
+			if( assign !== null ) {
+				parentValues[ assign ] = mout.string.ltrim( content );
```

Comment:
just thinking out loud, if `content` has multiple lines is the `mout.string.ltrim` gonna be enough? Maybe I'm seeing this wrong.